bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -74,16 +89,80 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     return py_arrays
 
 
+# Pythonizations for the following methods:
+# - ROOT.RDataFrame.Histo1D
+# - ROOT.RDataFrame.Histo2D
+# - ROOT.RDataFrame.Histo3D
+# - ROOT.RDataFrame.Profile1D
+# - ROOT.RDataFrame.Profile2D
+# Since a lot of their overloads take a ROOT.RDF.T*$DModel (where * can be
+# 'H' and 'Profile', while $ 1, 2 and 3) object as first argument,
+# we implement the possibility to pass the arguments of
+# ROOT.RDF.TH*DModel as a tuple in the first
+# argument of the five methods mentioned before.
+def pythonize_hp(self, partialmethod_list, *args):
+    # self: instantiation of RDataFrame
+    # partialmethod_list: list containing e.g. '_OriginalHisto1D' and
+    # e.g. the method Histo1D itself
+    # args: arguments passed by the user when he calls e.g Histo1D
+
+    # get the "original" method of the RDataFrame instantiation
+    original_function_hp = getattr(self, partialmethod_list[0])
+    # get the T*$DModel method
+    model_function = partialmethod_list[1]
+
+    if type(args[0]) == tuple:
```

Comment:
You need to check for the case in which the user (erroneously) calls the method with no arguments. In that case, you just forward the call to the original method (which will fail accordingly but more meaningfully).

Reply:
Also, for this line, instead of `if type(args[0]) == tuple`, I would use the more common:
```
first_arg = args[0]
if isinstance(first_arg, tuple):
```